blog/content/post/2020-04-29-trenchboot-uefi-environment.md

```diff
@@ -0,0 +1,821 @@
+---
+title: Installing Trenchboot in UEFI environments
+abstract: This blog post will show you how to install NixOS on UEFI platforms
+          and how to install Trenchboot on them.
+          
+cover: /covers/trenchboot-logo.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2020-04-29
+archives: "2020"
+
+tags:
+  - firmware
+  - security
+  - open-source
+categories:
+  - Firmware
+  - OS Dev
+  - Security
+
+---
+
+If you haven't read previous blog posts about TrenchBoot project, I encourage
+to do so. We have presented our motivation, project's overview concept and
+goals there. If you want to keep up with project development, stay tuned and
+always try to catch up missed articles.
+
+> Those articles will not always be strictly technical. Therefore, if you think
+some topics should be elaborated, feel free to ask us a question.
+
+In this article, I will explain how to install the NixOS on machines with UEFI
+firmware and hwo to install TrenchBoot. The most essential parts of system will
```

Comment:
```suggestion
firmware and how to install TrenchBoot. The most essential parts of the system will
```

Spellchecke  - I will not add more suggestions I assume you will do it anyway at the end

Reply:
Applied spellcheck